components/Card/index.js

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import styled from "styled-components";
+
+const StyledCard = styled.div`
+  border: 1px solid #ddd;
+  border-radius: 10px;
+  padding: 20px;
+  margin: 20px;
```

Comment:
I think the margin here could likely be replaced by adding some layouting to a component containing the cards. Perhaps a `CardsContainer` component on your ìndex.js` might be beneficial.

Reply:
We fixed that by making the ProjectList a flexbox, you can see that in our new pull request